pom.xml

```diff
@@ -24,11 +22,10 @@
     <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Jenkins Sauce OnDemand plugin</name>
-    <description>A Jenkins plugin to help create and teardown Sauce Labs tests and Sauce Connect sessions</description>
```

Comment:
Moved to `src/main/resources/index.jelly` in a recent version of Maven HPI plugin.